ansible_collections/arista/cvp/plugins/module_utils/resources/schemas/v3.py

```diff
@@ -192,16 +192,13 @@
                     },
                     "imageBundle": {
                         "$id": "#/items/anyOf/0/properties/imageBundle",
-                        "type": "array",
-                        "title": "The imageBundle schema",
+                        "type": "string",
```

Comment:
If this is already in use anywhere, this change would be breaking, correct?

Reply:
Also make sure to add this imageBundle key to any schema where the same data set might be exposed. Like we saw recently, the same data set is passed around so all the schemas has to take everything into account.

Reply:
Correct - but it wasn't actually used anywhere before

Reply:
To colin's point, this was not exposed in the docs so maybe no one is using this unless they went looking for it in the code, but to Claus's point, this is a breaking change. @colinmacgiolla can we make this work without changing the type from `array` to `string` ?

Reply:
Just to document here, in line with the discussions around #508 we will leave the type as `string`